french/dessine/caramel/mangez-caramel.md

```diff
@@ -0,0 +1 @@
+Vous mangez la porte de caramel avec du pain. Après quelques mintues, vous voyez la lumière. Vous voyez aussi... un lapin?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.